selene_sdk/predict/model_predict.py

```diff
@@ -866,9 +907,9 @@ def variant_effect_prediction(self,
             center = pos + len(ref) // 2
             start = center - self._start_radius
             end = center + self._end_radius
-
-            seq_encoding = self.reference_sequence.get_encoding_from_coords(
-                chrom, start, end, strand=strand)
+            sequence = self.reference_sequence.get_sequence_from_coords(chrom, start, end, strand=strand)
```

Comment:
since we do these 3 steps so often in these methods (get sequence, seq to enc, UNK_BASE in seq), I wonder if this should be moved to a helper function. just a suggestion